types/tablet_v2/wlr_tablet_v2_tool.c

```diff
@@ -226,11 +226,11 @@ struct wlr_tablet_v2_tablet_tool *wlr_tablet_tool_create(
 
 	tool->wlr_tool = wlr_tool;
 	wl_list_init(&tool->clients);
+	wl_list_init(&tool->surface_destroy.link);
```

Comment:
Note for reviewer: this field already existed in the struct, but as far as I can tell was never used.